egs/librispeech/ASR/transducer_emformer/streaming_decode.py

```diff
@@ -333,7 +344,15 @@ def greedy_search(
     blank_id = model.decoder.blank_id
     context_size = model.decoder.context_size
     device = model.device
+    assert len(streams) == encoder_out.size(0)
+    assert encoder_out.ndim == 3
 
+    for s in streams:
+        if s.hyp is None:
+            s.hyp = Hypothesis(
```

Comment:
For greedy search, `s.hyp` can be a simple list.
If you do want to use `Hypothesis`, please also update its `log_prob` in the following computation.